terraform/load-balancing.tf

```diff
@@ -43,6 +43,51 @@ resource "aws_lb_listener" "validator-ac" {
   }
 }
 
+resource "aws_lb" "json-rpc" {
+  name                             = "${terraform.workspace}-jsonrpc"
+  load_balancer_type               = "application"
+  enable_cross_zone_load_balancing = true
+  subnets                          = aws_subnet.testnet.*.id
+  security_groups                  = [aws_security_group.validator.id]
+}
```

Comment:
need to add "dualstack"  here, for AAAA record to work.

Reply:
@thefallentree sure

Reply:
will do this in a separate PR